### PR TITLE
Fix/button group qa review

### PR DESCRIPTION
**Pull request by juanmaguitar**
QA Feedback implemented:
1- Secondary full width: No ajusta su anchura en mobile (he probado en desktop responsive, iphone y android)    
2- Tertiary y tertiary negative: Estaría bien que todos los buttons tengan el mismo border radius en todos sus lados. (Quizás de esta manera se podrían reaprovechar estos buttons groups para el componente pagination..)

### CODE REVIEW

File: components/atom/button/src/index.scss [hunk 1]
```diff
@@ -86,21 +86,30 @@ $p-atom-button-large: $p-l !default;
   }
 
   &-group {
+    margin-left: -1px;
+    margin-right: 0px;
```
Comment (jcger) [inline components/atom/button/src/index.scss]: there should be no unit if the value is 0

File: components/atom/button/src/index.scss [hunk 2]
```diff
@@ -86,21 +86,30 @@ $p-atom-button-large: $p-l !default;
...
     
     & + & {
       margin-left: -1px;
```
Comment (davidbarna) [inline components/atom/button/src/index.scss]: Shouldn't it be `margin-left: -$bdrs-m;` ?
You should try to put $bdrs-m: 3px in your dev environment and have a look on how it behaves.

File: demo/molecule/buttonGroup/demo/index.scss [hunk 3]
```diff
@@ -4,6 +4,14 @@
 
   &-section {
     margin: 10px;
+    
+    &--fullWidth {
+      border: 2px dashed blue;
+      overflow: hidden;
+      padding: 10px;
```
Comment (davidbarna) [inline demo/molecule/buttonGroup/demo/index.scss:11]: No absolute values for marings, padding, etc...

File: demo/molecule/buttonGroup/demo/index.scss [hunk 4]
```diff
@@ -4,6 +4,14 @@
 
   &-section {
     margin: 10px;
+    
+    &--fullWidth {
+      border: 2px dashed blue;
```
Comment (davidbarna) [inline demo/molecule/buttonGroup/demo/index.scss:9]: MAke use this border is not relative to buttons border values

**Review by davidbarna — CHANGES_REQUESTED** [review]

File: demo/molecule/buttonGroup/demo/index.scss (hunk 4 above)
Comment (juanmaguitar) [reply demo/molecule/buttonGroup/demo/index.scss:9]: this is just some border style for the demo

File: demo/molecule/buttonGroup/demo/index.scss (hunk 3 above)
Comment (juanmaguitar) [reply demo/molecule/buttonGroup/demo/index.scss:11]: these are just some styles for the demo

File: components/atom/button/src/index.scss (hunk 2 above)
Comment (juanmaguitar) [reply components/atom/button/src/index.scss]: Not really, this value achieves the needed effect of grouping the buttons while respecting original dimensions on each button

File: demo/molecule/buttonGroup/demo/index.scss (hunk 3 above)
Comment (davidbarna) [reply demo/molecule/buttonGroup/demo/index.scss:11]: Sorry !!! 🤦‍♂️

**Review by davidbarna — DISMISSED** [review]

**Review by jcger — APPROVED** [review]

**Review by midudev — APPROVED** [review]

**Pull request merged**